lib/suseconnect_register.pm

```diff
@@ -108,6 +114,19 @@ sub is_module {
     return 1 if (grep { $addon eq $_ } qw(asmm contm lgm pcm tcm wsm idu ids));
 }
 
+# De-register the system from the SUSE Customer Center
+sub suseconnect_deregistration {
```

Comment:
maybe better just to move this code into scc_derigistration ? And people will just loadtest when they need this ?

Reply:
+1